docs/examples/values/prod/bitbucket.yaml

```diff
@@ -0,0 +1,349 @@
+    secretName: bitbucket-db-secret
+    # -- The key in the Secret used to store the database login username
+    usernameSecretKey: username
+    # -- The key in the Secret used to store the database login password
+    passwordSecretKey: password
+
+bitbucket:
+  service:
+    # -- The port on which the Bitbucket Kubernetes service will listen
+    port: 80
+    # -- The type of Kubernetes service to use for Bitbucket
+    type: ClusterIP
+    annotations: {}
+  # -- Enable or disable an additional service for exposing SSH for external access.
+  # Disable when the SSH service is exposed through the ingress controller, or enable if the ingress controller does
+  # not support TCP.
+  sshService:
+    enabled: false
+    # -- Port to expose the SSH service on.
+    port: 22
+    type: LoadBalancer
+    # -- Annotations for the SSH service. Useful if a load balancer controller needs extra annotations.
+    annotations: {}
+  # -- Enable or disable security context in StatefulSet template spec. Enabled by default with UID 2003.
+  # -- Disable when deploying to OpenShift, unless anyuid policy is attached to service account
+  securityContext:
+    enabled: true
+    # -- The GID used by the Bitbucket docker image
+    gid: "2003"
+  ports:
+    http: 7990
+    ssh: 7999
+    hazelcast: 5701
+
+  license:
+    # -- The name of the Kubernetes Secret that contains the Bitbucket license key.
+    # If specified, then the license will be automatically populated during Bitbucket setup.
+    # Otherwise, it will need to be provided via the browser after initial startup.
+    secretName: bitbucket-license
+    # -- The key in the Kubernetes Secret that contains the Bitbucket license key
+    secretKey: license-key
+
+  sysadminCredentials:
+    # -- The name of the Kubernetes Secret that contains the Bitbucket sysadmin credentials
+    # If specified, then these will be automatically populated during Bitbucket setup.
+    # Otherwise, they will need to be provided via the browser after initial startup.
+    secretName: bitbucket-sysadmin-creds
+    # -- The key in the Kubernetes Secret that contains the sysadmin username
+    usernameSecretKey: username
+    # -- The key in the Kubernetes Secret that contains the sysadmin password
+    passwordSecretKey: password
+    # -- The key in the Kubernetes Secret that contains the sysadmin display name
+    displayNameSecretKey: displayName
+    # -- The key in the Kubernetes Secret that contains the sysadmin email address
+    emailAddressSecretKey: emailAddress
+
+  clustering:
+    # -- Set to true if Data Center clustering should be enabled
+    # This will automatically configure cluster peer discovery between cluster nodes.
+    enabled: true
+
+  elasticSearch:
+    # -- The base URL of the external ElasticSearch instance to be used.
+    # If this is defined, then Bitbucket will disable its internal ElasticSearch instance.
+    baseUrl:
```

Comment:
Elasticsearch is quite important part, we should provide example for it.

Reply:
Yes, I think we have just forgotten to add it to prerequisites. I think it is quite an important part of the BB experience.